### ISSUE TITLE
Automate Go and Fyne update

### ISSUE THREAD

**Issue by Bluebugs** [fyne-io/fyne-cross-images#38]
It would relieve the amount of work required to maintain this repository if we could also depends on some automation to automatically update our Fyne and Go version. General idea would be to have a github action for each. For Fyne, it could poke github API to figure out the latest release with a tag of the form (vX.Y.Z) and update base/Dockerfile:FYNE_VERSION accordingly. For Golang, it is a bit more complex, as we will need to figure out the latest release from tag (similar to Fyne), but then lookup on the website what are the expected value for checksum of the relevant linux arm64 and linux amd64 package.

**Comment by andydotxyz** [comment]
I'm not sure that auto-updating versions is a good idea for something that automatically pushes out to a repository that development tooling pulls from.
However simplifying would be great, maybe a script or repo that manages the Fyne-cross setup would help?
Bearing in mind that over time the main tooling will be moving to a command line that must be very robust as it is core to the project...

**Comment by Jacalz** [comment]
Automatic PRs that updates Go and Fyne but still require manual merge could be an option

**Comment by Bluebugs** [comment]
Indeed, I was thinking of something like what dependabot does. Generate a pr that can be reviewed appropriately.

**Comment by andydotxyz** [comment]
Ah cool. Automating the preparation of an update is fine, automatically updating not so much.

**Comment by lorypelli** [comment]
I was also going to ask for version updates, it should be added for sure...

**Comment by Bluebugs** [comment]
The goal of this issue is to figure out automation, not doing it manually :-)

**Comment by lorypelli** [comment]
> The goal of this issue is to figure out automation, not doing it manually :-)

Yeah, but automation wasn't added yet, it should be added because it's better, but in the meantime, I made PR
